### ISSUE TITLE
use deep selector rather than id for perseus styling

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri-exercise-perseus-plugin#125]
we have two uses of IDs:

* https://github.com/learningequality/kolibri-exercise-perseus-plugin/blob/master/kolibri_exercise_perseus_plugin/assets/src/views/index.vue#L5
* https://github.com/learningequality/kolibri-exercise-perseus-plugin/blob/master/kolibri_exercise_perseus_plugin/assets/src/views/index.vue#L36

if possible, we should remove all ids and instead leverage vue's deep selectors.

**Comment by rtibbles** [comment]
Code lines you link to are different on release-v0.7.x branch, so I think this might already have been resolved by #123?

**Comment by indirectlylit** [comment]
No, in fact it was that PR which brought it to my attention. Note it still uses an id: https://github.com/learningequality/kolibri-exercise-perseus-plugin/pull/123/files#diff-0730f270ce8c448923cd937eff3d0c79R441

**Comment by indirectlylit** [comment]
(cc @christianmemije)

**Comment by rtibbles** [comment]
I guess I'm not quite understanding why a deep selector is preferable when it is applying styling specifically to that div and its children?

**Comment by indirectlylit** [comment]
Ah, I thought that this was a case where we were using an unscoped style block, but [thats not the case](https://github.com/learningequality/kolibri-exercise-perseus-plugin/pull/123/files#diff-0730f270ce8c448923cd937eff3d0c79R433). So you're right a deep selector isn't necessary.

That said, classes should be used in stead of IDs here because an ID ought to be unique on a page, which wouldn't be guaranteed if someone dropped multiple perseus renderers on the same page.

I'll update the issue
